tutorials/tts/4_TTS_FastPitch_Finetuning.ipynb

```diff
@@ -0,0 +1,528 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "id": "8d0bbac2",
+   "metadata": {},
+   "source": [
+    "# Finetuning FastPitch for a new speaker"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "2d063299",
+   "metadata": {},
+   "source": [
+    "In this tutorial, we will finetune a single speaker FastPitch (with alignment) model on limited amount of new speaker's data. We cover two finetuning techniques: \n",
+    "1. We finetune the model parameters only on new speaker's text and speech pairs; \n",
+    "2. We add a learnable speaker embedding layer to the model, and finetune on a mixture of original speaker's and new speaker's data.\n",
+    "\n",
+    "We will first prepare filelists containing the audiopaths and text of the samples on which we wish to finetune the model, then generate and run a training command to finetune Fastpitch on 5 mins of data, and finally synthesize the audio from the trained checkpoint."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "2502cf61",
+   "metadata": {},
+   "source": [
+    "## Creating filelists for training"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "81fa2c02",
+   "metadata": {},
+   "source": [
+    "We will first create filelists of audio on which we wish to finetune the FastPitch model. We will create two kinds of filelists, one which contains only the audio files of the new speaker and one which contains the mixed audio files of the new speaker and the speaker used for training the pre-trained FastPitch Checkpoint."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "b7b1563d",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import random\n",
+    "import os\n",
+    "import json\n",
+    "import torch\n",
+    "import IPython.display as ipd\n",
+    "from matplotlib.pyplot import imshow\n",
+    "from matplotlib import pyplot as plt\n",
+    "\n",
+    "data_dir = <ADD_PATH_TO_DIRECTORY_CONTAINING_HIFIGAN_DATASET> # Download dataset id 78419 from NGC. Specify path to Hi_Fi_TTS_v_0\n",
```

Comment:
`Download dataset id 78419 from NGC`: This is not available to the public. Can you point them to OpenSLR instead: https://www.openslr.org/109/